### PR TITLE
@snow SNOW-633432 Parquet file generator (fixed dep)

### PR DESCRIPTION
**Pull request by sfc-gh-azagrebin**
reopen #200 again after bumping Hadoop to the latest version 3.3.4.

### CODE REVIEW

**Review by sfc-gh-gdoci — APPROVED** [review]
lgtm, thanks!

**Review by sfc-gh-tjones — APPROVED** [review]
LGTM assuming it's just a re-open w/ a version change.

File: src/main/java/net/snowflake/ingest/streaming/internal/BlobMetadata.java [hunk 1]
```diff
@@ -49,9 +49,16 @@ List<ChunkMetadata> getChunks() {
...
-  //    return bdecVersion.toByte();
-  //  }
+  /**
+   * Create {@link BlobMetadata} in case of {@link Constants.BdecVersion#ONE} and {@link
+   * BlobMetadataWithBdecVersion} otherwise to send BDEC version to server side.
+   */
+  static BlobMetadata createBlobMetadata(
+      String path, String md5, Constants.BdecVersion bdecVersion, List<ChunkMetadata> chunks) {
+    // TODO SNOW-659721: Unify BlobMetadata with BlobMetadataWithBdecVersion once server side
+    // BdecVersion in production
```
Comment (sfc-gh-tzhang) [inline src/main/java/net/snowflake/ingest/streaming/internal/BlobMetadata.java:59]: Does the server side change in prod now?

**Review by sfc-gh-tzhang — APPROVED** [review]
LGTM! Thanks @sfc-gh-azagrebin and @sfc-gh-gdoci for driving this to completion, hopefully we won't get any complains from Dependabot or Snyk :)

**Pull request merged**